react/src/MultiSelect/components/MultiDropdownItem/MultiDropdownItem.tsx

```diff
@@ -39,6 +47,7 @@ export const MultiDropdownItem = ({
   return (
     <ListItem
       sx={styles.item}
+      data-active={dataAttr(isSelected)}
```

Comment:
This is causing the active styling to be set on the multidropdownitem, which is a regression. i don't think we need this prop here

Reply:
o ye, thanks! fixed in [a685fc2](https://github.com/opengovsg/design-system/pull/252/commits/a685fc25219c6676d8b1fc70a08a42a6f162434f)